proto/nym/vpn.proto

```diff
@@ -550,21 +550,63 @@ message RemoveAccountResponse {
   AccountError error = 2;
 }
 
-message GetLocalAccountStateRequest {}
+enum MnemonicState {
+  MNEMONIC_STATE_UNSPECIFIED = 0;
+  MNEMONIC_STATE_NOT_STORED = 1;
+  MNEMONIC_STATE_STORED = 2;
+}
 
-message GetLocalAccountStateResponse {
-  string json = 1;
+enum AccountState {
+  ACCOUNT_STATE_UNSPECIFIED = 0;
+  ACCOUNT_STATE_NOT_REGISTERED = 1;
+  ACCOUNT_STATE_INACTIVE = 2;
+  ACCOUNT_STATE_ACTIVE = 3;
+  ACCOUNT_STATE_DELETE_ME = 4;
+}
+
+enum SubscriptionState {
+  SUBSCRIPTION_STATE_UNSPECIFIED = 0;
+  SUBSCRIPTION_STATE_NOT_REGISTERED = 1;
+  SUBSCRIPTION_STATE_PENDING = 2;
+  SUBSCRIPTION_STATE_ACTIVE = 3;
+  SUBSCRIPTION_STATE_COMPLETE = 4;
+}
+
+enum DeviceState {
+  DEVICE_STATE_UNSPECIFIED = 0;
+  DEVICE_STATE_NOT_REGISTERED = 1;
+  DEVICE_STATE_INACTIVE = 2;
+  DEVICE_STATE_ACTIVE = 3;
+  DEVICE_STATE_DELETE_ME = 4;
```

Comment:
That is a protoc compile fail since those are global and clashes in some languages (e.g C++)

Reply:
> That is a protoc compile fail since those are global and clashes in some languages (e.g C++)

![image](https://github.com/user-attachments/assets/e516ad45-fb11-44b5-86b6-69a820ff00f1)

Reply:
if you mean something like

```
//top level of proto
enum DeviceState {
  DEVICE_STATE_UNSPECIFIED = 0;
  NOT_REGISTERED = 1;
  INACTIVE = 2;
  ACTIVE = 3;
  DELETE_ME = 4;
}

//top level of proto
enum AnotherState {
  ANOTHER_STATE_UNSPECIFIED = 0;
  NOT_REGISTERED = 1;
  INACTIVE = 2;
  ACTIVE = 3;
  DELETE_ME = 4;
}
```
it compiles without issue too

Reply:
```
❯ cat test.proto 
syntax = "proto3";

package tmp;

message Empty {}

enum State1 {
    ACTIVE = 1;
}

enum State2 {
    ACTIVE = 1;
}

message Response {
    State1 state = 1;
}

service NymVpnd {
    rpc GetState(Empty) returns (Response) {}
}
/tmp                                                                                                                                                   
❯ ~/Downloads/protoc/bin/protoc -o /tmp/test test.proto
test.proto:12:5: "ACTIVE" is already defined in "tmp".
test.proto:12:5: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "ACTIVE" must be unique within "tmp", not just within "State2".
/tmp                                                                                                                                                   
❯ ~/Downloads/protoc/bin/protoc --version                     
libprotoc 28.3
```

Reply:
Oh yes I can reproduce now
Don't know why but my previous proto snippet somehow compiled, but not anymore and I get the same errors you shared above

Reply:
ok well 2cents propositions

```
enum AccountState {
  ACCOUNT_STATE_UNSPECIFIED = 0;
  A_NOT_REGISTERED = 1;
  A_INACTIVE = 2;
  A_ACTIVE = 3;
  A_DELETE_ME = 4;
}

enum SubscriptionState {
  SUBSCRIPTION_STATE_UNSPECIFIED = 0;
  S_NOT_REGISTERED = 1;
  …
}

enum DeviceState {
  DEVICE_STATE_UNSPECIFIED = 0;
  D_NOT_REGISTERED = 1;
  …
}
```